doc/environment.yml

```diff
@@ -0,0 +1,20 @@
+name: qinfer
+dependencies:
+  - cython
+  - numpy
+  - scipy
+  - scikit-learn
+  - ipython
+  - future>=0.15
+  - matplotlib
+  - pytest
+  - pytest-cov
+  - pytest-xdist
+  - coveralls
+  - pylint
+  - pip:
+    # We also add additional tools for plotting.
+    - mpltools
+    # We use the PyPI version of QuTiP here since the conda version
+    # is on the wrong channel for use with RTD.
+    - qutip
```

Comment:
Maybe you know, but you can set the channels at the top of the yml under  a ``channels:`` title. But I'm not familiar with this particular qutip quirk...no need to change it if it works

Reply:
I've used that in other places, but for some reason I was under the mistaken impression that ReadTheDocs didn't support `channels:`. I'll investigate that soon.

Reply:
Ooh, I have no idea what RTD accepts! I was just thinking about conda env.